api/src/main/java/jakarta/persistence/criteria/CriteriaBuilder.java

```diff
@@ -702,7 +725,51 @@
      */	
     Expression<Double> sqrt(Expression<? extends Number> x);
 
-	
+    /**
+     * Create an expression that returns the exponential
+     * of its argument.
```

Comment:
Put a note, that base is `e` so it computes `e^x`.

Reply:
See: 6b986e8ccb20e808ac52783f3e4ba645a20ffe25

I've also clarified the semantics of `mod()`, since "modulus" is also used to mean the absolute value.

Reply:
OK, thanks.